bcgov/biohubbc
# BHBC-2282: Fix draft load

(https://quartech.atlassian.net/browse/BHBC-2282)

Testing procedure:
Baseline:
1. Log into Dev
2. Throttle your network
3. Load a draft project
4. You will see an empty draft project

As a result of this code change:
1. Go to the app link for this PR
2. Log in
3. Throttle your network
4. Load a draft project
5. You will see the content.
